test/evergreen.yml

```diff
@@ -57,10 +57,21 @@ functions:
   "fetch mongo repo" :
     command: shell.exec
     params:
+      shell: bash
       script: |
         set -o errexit
         set -o verbose
-        git clone https://github.com/mongodb/mongo
+        mongo_repo=https://github.com/mongodb/mongo
+        if [[ "${branch_name}" == "mongodb-5.0" ]]; then
+          git clone $mongo_repo -b v5.0
+        elif [[ "${branch_name}" == "mongodb-4.4" ]]; then
+          git clone $mongo_repo -b v4.4
+        elif [[ "${branch_name}" =~ "mongodb-" ]]; then   
+          echo "Checkout the correct mongodb branch for wiredtiger branch ${branch_name}"
```

Comment:
Thanks for the change!
Maybe reword the `echo` text as `'fetch mongo repo' does not currently handle branch ${branch_name}` and we're good to go.

Reply:
Yep, I agree

Reply:
Updated in [1e74f3a](https://github.com/wiredtiger/wiredtiger/pull/7603/commits/1e74f3a22e65144438bc1e480679b3426233503d)